docs/querying/sql-functions.md

```diff
@@ -1227,35 +1269,146 @@ Returns a number for each output row of a groupBy query, indicating whether the
 
 ## HLL_SKETCH_ESTIMATE
 
-`HLL_SKETCH_ESTIMATE(expr, [round])`
+Returns the distinct count estimate from an HLL sketch. The `expr` argument must return an HLL sketch. The optional `round` boolean argument rounds the estimate if set to true, with a default of false.
```

Comment:
```suggestion
Returns a distinct count estimate from an HLL sketch. To round the distinct count estimate, set `round` to true. Otherwise, `round` defaults to false.
```
`expr` is a parameter, not an argument. It doesn't return anything. 
It's obvious from the function syntax that `round` is optional.